org.eclipse.jdt.core/compiler/org/eclipse/jdt/internal/compiler/lookup/TypeSystem.java

```diff
@@ -98,6 +98,9 @@ public void swapUnresolved(UnresolvedReferenceBinding unresolvedType, ReferenceB
 					ReferenceBinding enclosing = resolvedType.enclosingType();
 					if (enclosing != null) {
 						this.enclosingType = resolvedType.isStatic() ? enclosing : (ReferenceBinding) env.convertUnresolvedBinaryToRawType(enclosing); // needed when binding unresolved member type
+						if(this.enclosingType.getClass() == ParameterizedTypeBinding.class) {
```

Comment:
May I ask for a "Ctrl + F" on selection? The code change doesn't look aligned with other formatting.

Reply:
done.